install_config/syncing_groups_with_ldap.adoc

```diff
@@ -860,3 +867,277 @@ users: <5>
+member: cn=Jane,ou=users,dc=example,dc=com
+member: cn=otheradmins,ou=groups,dc=example,dc=com
+
+dn: cn=otheradmins,ou=groups,dc=example,dc=com <2>
+objectClass: group
+cn: otheradmins
+owner: cn=admin,dc=example,dc=com
+description: Other System Administrators
+memberOf: cn=admins,ou=groups,dc=example,dc=com <1> <3>
+member: cn=Jim,ou=users,dc=example,dc=com
+----
+<1> The user's and group's memberships are listed as attributes on the object.
+<2> The groups are first-class entries on the LDAP server.
+<3> The *otheradmins* group is a member of the *admins* group.
+====
+
+To sync this group, you must first create the configuration file. The augmented
+Active Directory with recursive members schema requires you to provide an LDAP
+query definition for both user entries and group entries, as well as the attributes
+with which to represent them in the internal {product-title} Group records.
+Furthermore, certain changes are required in this configuration:
+
+- The `oadm groups sync` command must explicitly
+xref:../install_config/syncing_groups_with_ldap.adoc#running-ldap-sync[whitelist] groups.
+- The user's `groupMembershipAttributes` must specify
+https://msdn.microsoft.com/en-us/library/aa746475(v=vs.85).aspx[LDAP_MATCHING_RULE_IN_CHAIN].
+- The `groupUIDAttribute` must be set to `dn`.
+- The `groupsQuery`:
+  * Must not set `filter`.
+  * Must set a valid `derefAliases`.
+  * Should not set `baseDN` as that value is ignored.
+  * Should not set `scope` as that value is ignored.
+
+For clarity, the Group you create in {product-title} should use attributes other
+than the distinguished name whenever possible for user- or administrator-facing
+fields. For example, identify the users of a Group by their e-mail,
+and use the name of the Group as the common name. The following configuration
+file creates these relationships.
+
+.LDAP Sync Configuration Using Augmented Active Directory Schema With Recursive Members: *_augmented_active_directory_config_recursive.yaml_*
+====
+[source,yaml]
+----
+kind: LDAPSyncConfig
+apiVersion: v1
+url: ldap://LDAP_SERVICE_IP:389
+insecure: true
+augmentedActiveDirectory:
+    groupsQuery: <1>
+        derefAliases: never
+        pageSize: 0
+    groupUIDAttribute: dn <2>
+    groupNameAttributes: [ cn ] <3>
+    usersQuery:
+        baseDN: "ou=users,dc=example,dc=com"
+        scope: sub
+        derefAliases: never
+        filter: (objectclass=inetOrgPerson)
+        pageSize: 0
+    userNameAttributes: [ mail ] <4>
+    groupMembershipAttributes: [ "memberOf:1.2.840.113556.1.4.1941:" ] <5>
+----
+<1> `groupsQuery` filters cannot be specified.  The `groupsQuery` base DN and scope values are ignored.
+`groupsQuery` must set a valid `derefAliases`.
+<2> The attribute that uniquely identifies a group on the LDAP server. It must be set to `dn`.
+<3> The attribute to use as the name of the Group.
+<4> The attribute to use as the name of the user in the {product-title} Group record.
```

Comment:
Not sure if mail or sAMAccountName would be preferred in most installations... can at least note sAMAccountName as an alternative